dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/AbstractEventService.java

```diff
@@ -623,7 +623,7 @@ public Event getEvent( ProgramStageInstance programStageInstance, boolean isSync
         if ( ou != null )
         {
             event.setOrgUnit( ou.getUid() );
-            event.setOrgUnitName( ou.getName() );
+            event.setOrgUnitName( ou.getDisplayName() );
         }
```

Comment:
Why here do we add the `displayName` but later we do 
`coalesce(ou_displayname.value, ou.name) as ou_name`